### PR TITLE
Add CVE-2019-5715 in silverstripe/framework

### PR DESCRIPTION
**Pull request by robbieaverill**

### CODE REVIEW

**Comment by robbieaverill** [comment]
I've removed the symbiote/silverstripe-multivaluefield report, since it's actually not a security vulnerability as such but broken functionality caused by the fix in silverstripe/framework

**Comment by fabpot** [comment]
Thank you

File: silverstripe/framework/CVE-2019-5715.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+title:     'CVE-2019-5715: Reflected SQL Injection through Form and DataObject'
+link:      https://www.silverstripe.org/download/security-releases/ss-2018-021
+cve:       CVE-2019-5715
+branches:
+    3.6.x:
+        time:     2019-02-19 11:55:05
+        versions: ['>=3.6.0', '<3.6.7']
```
Comment (stof) [inline silverstripe/framework/CVE-2019-5715.yaml]: this should be `>=3.0.0` instead, as it says `all versions of SilverStripe 3 prior to 3.6.7, 3.7.3`

Comment (robbieaverill) [reply silverstripe/framework/CVE-2019-5715.yaml]: Ok, I got confused by the branch name - will update anyway

Comment (robbieaverill) [reply silverstripe/framework/CVE-2019-5715.yaml]: See #367

**Pull request merged**